### ISSUE TITLE
low acc when training Bert-Large

### ISSUE THREAD

**Issue by shivakasu** [google-research/bert#288]
I tried both Bert-Base and Bert-Large on IMDB sentiment classification dataset.
Bert-Base got 92% eval-acc, while Bert-Large got 49% eval-acc.
Is there anything wrong in Bert-Large? or it only support large dataset?

**Comment by hellozjj** [comment]
？How did you fix this?

**Comment by shivakasu** [comment]
> ？How did you fix this?

Large model needs more training epochs....

**Comment by enod** [comment]
@shivakasu 
Were you be able to pass 95%(ULMFIT scored) accuracy by training longer?

**Comment by hellozjj** [comment]
> > ？How did you fix this?
> 
> Large model needs more training epochs....


How much accuracy do you  get  when using  Large model?

**Comment by PaulZhangIsing** [comment]
@shivakasu may I know how much time you spent on fine-tuning it?

**Comment by shivakasu** [comment]
> @shivakasu may I know how much time you spent on fine-tuning it?

I dont know how to use multi-gpu, so I trained Bert-Large on cpu. It took about one day for one epoch.

**Comment by HaoyuHu** [comment]
Maybe you should increase your batch size and learning rate.
For data-parallelism on GPUs(no horovod), this project may help you.
https://github.com/HaoyuHu/bert-multi-gpu

**Comment by shivakasu** [comment]
> Maybe you should increase your batch size and learning rate.
> For data-parallelism on GPUs(no horovod), this project may help you.
> https://github.com/HaoyuHu/bert-multi-gpu

666，thanks！
